src/_private/components/BarChart/helpers.ts

```diff
@@ -275,3 +275,34 @@ export const getCommonGroupsMaxColumns = (groups: readonly GroupItem[]) => {
     ...groups.map(group => Math.max(group.columns.length, group.reversedColumns.length))
   )
 }
+
+export const gapSpace = {
```

Comment:
[note]
Есть предложение сделать переменную больше похожей на константу:

```suggestion
export const GRID_GAP_SPACE = {
```

Reply:
Поправил